packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -53,8 +53,19 @@ export type AssetInfo = Asset & {
   location?: Location;
   exif?: object;
   isFavorite?: boolean; //iOS only
+  isNetworkAsset?: boolean; //iOS only
 };
 
+export interface AssetInfoQueryOptions {
```

Comment:
1. It should be a type — interfaces are intended for things that someone may make a class that implements them. We pass options as a plain object so a type fits more for its purposes.
2. Now we're trying to use the module name as a prefix for all types/enums/interfaces so they don't conflict with other modules, so I'd prefer `MediaLibraryAssetInfoQueryOptions` 😉